api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -543,6 +554,19 @@ public void saveGlobalProperty_shouldOverwriteGlobalPropertyIfExists() {
 		Assert.assertEquals("new-even-more-correct-value", as.getGlobalProperty("a_valid_gp_key"));
 	}
 	
+	/**
+	 * @see AdministrationService#saveGlobalProperty(GlobalProperty)
+	 */
+	@Test
+	public void saveGlobalProperty_shouldReturnSavedGlobalProperty() {
```

Comment:
this test exists already at `saveGlobalProperty_shouldCreateGlobalPropertyInDatabase`, so please remove it

how did you figure out that this was needed?
please generate coverage locally, this will help you decide whats missing

Reply:
@teleivo Coveralls showed a red line in the return statement, so I guessed it has not been tested.